src/back/glsl/mod.rs

```diff
@@ -2122,6 +2117,17 @@ impl<'a, W: Write> Writer<'a, W> {
             uniforms,
         })
     }
+
+    /// Helper method used to produce GL specific unique name
+    /// GL identifiers starting with "gl_" are not allowed
+    fn get_name(&self, name_key: &NameKey) -> String {
```

Comment:
instead of handling it here, can we instead add support for "reserved prefixes" to `Namer` itself?

Reply:
Done!